lib/adapters/linter-push-v2-adapter.ts

```diff
@@ -113,13 +113,13 @@ export default class LinterPushV2Adapter {
     if (path != null) {
       const diagnosticCodes = this._diagnosticCodes.get(path);
       if (diagnosticCodes != null) {
-        return diagnosticCodes.get(getCodeKey(range, text));
+        return diagnosticCodes.get(getCodeKey(range, text)) || null;
       }
     }
     return null;
   }
 }
 
 function getCodeKey(range: atom.Range, text: string): string {
-  return [].concat(...range.serialize(), text).join(',');
+  return ([] as any[]).concat(...range.serialize(), text).join(',');
```

Comment:
A little hack to make the type system happy

Reply:
I wonder if we should just ``${range.serialize()},${text}`` here?

Reply:
I'll give it a shot!

Reply:
`range.serialize()` won't work because it returns an array of arrays, that's why the `[].concat()` with the spread operator is used.  It builds a flat array of items that can be joined.  I'll leave it as it is for now